.github/workflows/auto-merge-dependabot.yml

```diff
@@ -0,0 +1,48 @@
+name: Dependabot auto-merge
```

Comment:
@Junjiequan I was just wondering if we can use some of the ready made actions for this. For example like: https://github.com/marketplace/actions/dependabot-auto-merge

Reply:
We could. Initially I tried to use exactly this 3rd party action in scicat backend.. And I was being accused for using intransparent third-party action from one of the reviewers from the other facility : D
I don't know should we use this 3rd party action or just have our own version. 
One benefit of having our own flow is ofc we have more control.